crates/bevy_winit/src/lib.rs

```diff
@@ -612,9 +616,15 @@ pub fn winit_runner_with(mut app: App) {
             }
             Event::RedrawEventsCleared => {
                 {
-                    let winit_config = app.world.resource::<WinitSettings>();
-                    let windows = app.world.resource::<Windows>();
+                    let mut windows = app.world.resource_mut::<Windows>();
                     let focused = windows.iter().any(|w| w.is_focused());
+                    if !windows.iter().any(|w| w.visible()) {
```

Comment:
This doesn't seem right. I don't think this code should run every frame, but also this doesn't work if you have multiple windows. Try running the multiple_windows example to see what I mean

Reply:
If you remove the condition both windows will appear, but if you keep this condition only one window appears at a time